docs/_docs/refs/mojito-locales.md

```diff
@@ -115,12 +119,15 @@ permalink: /docs/refs/mojito-locales/
 | ms-BN	        | Malay (Brunei Darussalam) |
 | ms-MY	        | Malay (Malaysia) |
 | mt-MT	        | Maltese (Malta) |
+| mus-021	        | Creek (Northern America) |
 | nb-NO	        | Norwegian (Norway) |
 | nl-BE	        | Dutch (Belgium) |
 | nl-NL	        | Dutch (Netherlands) |
 | nn-NO	        | Norwegian (Nynorsk) (Norway) |
 | ns-ZA	        | Northern Sotho (South Africa) |
+| nv-003	        | Navajo (North America) |
 | ny-MW	        | Nyanja; Chewa; Chichewa (Malawi) |
+| oj-021	        | Ojibwa (Northern America) |
```

Comment:
Ojibway is a French term. Anishinaabe is what they call themselves

Reply:
ditto above

Quick research says this is much more entrenched than other terms. Seems to have originated in 2009 from days when changes happened via internet engineering taskforce (IETF) RFC:
- https://tools.ietf.org/html/draft-ietf-ltru-4645bis-10
- http://cldr.unicode.org/development/development-process/design-proposals/languages-to-show-for-translation

Reply:
Nothing related in tracker for:
- ojibwa https://unicode-org.atlassian.net/browse/ICU-3455?jql=text%20~%20%22ojibwa%22
- Anishinaabe https://unicode-org.atlassian.net/browse/CLDR-3797?jql=text%20~%20%22Anishinaabe%22

Reply:
Interesting. I will have to read up more on this. Bureaucracy but without engaging Native peoples it seems. Is that the norm?